src/test/scenarios/datafactory/output/src/datafactory-preview/azext_datafactory_preview/tests/latest/example_steps.py

```diff
@@ -77,8 +77,7 @@ def step_configure_factory_repo(test, rg, checks=None):
     test.cmd('az datafactory configure-factory-repo '
              '--factory-resource-id "/subscriptions/{subscription_id}/resourceGroups/{rg}/providers/Microsoft.DataFacto'
              'ry/factories/{myFactory}" '
-             '--factory-vsts-configuration account-name="ADF" collaboration-branch="master" last-commit-id="" '
-             'project-name="project" repository-name="repo" root-folder="/" tenant-id="" '
+             '--factory-vsts-configuration FactoryVSTSConfiguration project  ADF repo / master '
```

Comment:
I think if it's a string you need to add " " to let python know it's a string.

Reply:
Hi @qiaozha , "" added in latest push:)